skycoin/skywire
# UI for managing the reward addresses

Did you run `make format && make check`?
The go code was not changed. `npm run lint` and `npm run build` were used.

 Changes:	
- Now the hypervisor UI has options for adding and removing the reward addresses.

How to test this PR:
- On the visor list, press the menu button at the top-right part of the screen. There is an option for changing the address for all visors.
- On the visor details page, the current address is shown on the info bar at the right. There is a button for changing the address.

Note: for seing the changes, you must recompile the UI with `make build-ui`
